README.md

```diff
@@ -58,6 +58,15 @@ icons/
 
 ### Offline Maps
 
+Use [mapbox-style-downloader](https://github.com/digidem/mapbox-style-downloader) to download styles, tiles, glyphs, and sprites for offline use.
```

Comment:
Maybe link to the npm page, and add instructions to install globally with `npm install --global`.